GETTING_STARTED.md

```diff
@@ -186,6 +186,31 @@ Another advantage of this approach is that if you need to use a `DbContext` insi
 * `WebActivatorEx`'s `PreApplicationStartMethod` (the `PreStart` method in our sample above) runs **before** OWIN's Startup method.
 	* See this StackOverflow post: https://stackoverflow.com/questions/21462777/webactivatorex-vs-owinstartup
 
+### How to deal with `ObjectDataSource` controls
```

Comment:
I need to split-up the `GETTING_STARTED.md` file into smaller doc files under `/docs/*` to keep them more focused. I'll do that now and invite you to submit this as a new doc file there.